### ISSUE TITLE
QoL - Add shortcut options to position Icons

### ISSUE THREAD

**Issue by diggeddy** [tomusborne/generateblocks#943]
### Description
Add icon position shortcut in Icon Panel for Headline and Button.
4 primary positions should suffice:

Above
Below
Before
After

**Comment by leohsiang** [comment]
So the options would change the settings in the Layout panel? If so then what happens to the shortcut setting if users changed something in the Layout panel after? We are gonna work on some quick docs for this.

**Comment by diggeddy** [comment]
Yeah they would just be a shortcut mechanism that would set the flex options for you.
Of all the QoL improvements this is probably the lowest priority.

**Comment by leohsiang** [comment]
Still needs a bit tidy up but this should be helpful for now: https://docs.generateblocks.com/article/headline-icon-alignment/

**Comment by diggeddy** [comment]
Closing. Improve made in 2.0
